tests/golden/check-local-refs/dir1/dir2/d2f1.md

```diff
@@ -9,23 +9,32 @@
 [bad-cf-ref](#bad)
 
 # Relative links
-[existing-file-rel](d2f2.md)
-[existing-file-rel](./d2f2.md)
-[existing-file-rel](../dir2/.././d1f1.md)
+[existing-file-rel-1](d2f2.md)
+[existing-file-rel-2](./d2f2.md)
+[existing-file-rel-3](../dir2/.././d1f1.md)
+[existing-file-rel-4](d2f3.yaml)
 
+[existing-dir-rel-1](..)
+[existing-dir-rel-2](../dir2)
+[existing-dir-rel-3](../dir2/)
 
 [existing-anchor-rel-1](d2f2.md#existing-anchor-d2f2)
 [existing-anchor-rel-2](./d2f2.md#existing-anchor-d2f2)
 [existing-anchor-rel-3](../dir2/../d1f1.md#existing-anchor-d1f1)
 
 [bad-file-rel](../a/b/c/unexisting-file.md)
+[bad-slash-file-rel](d2f2.md/)
```

Comment:
This link should be valid, both github and gitlab correctly link to `[a](./README.md/)`. I'll create an issue for this.

Reply:
Ok, I left this untouched